.babelrc

```diff
@@ -1,9 +1,17 @@
 {
-  "presets": ["es2015"],
-  "plugins": [
-    "transform-flow-strip-types",
-    "transform-es3-property-literals",
-    "transform-es3-member-expression-literals",
-    ["transform-es2015-for-of", {"loose": true}]
-  ]
+  "plugins": ["transform-flow-strip-types"],
+  "env": {
+    "production":{
+      "presets": [["es2015", {"modules": false}]]
+    },
+    "test": {
+      "presets": ["es2015"],
```

Comment:
`modules: false` is just to allow webpack to transpile `import/export` rather than babel. Tree shaking is a secondary benefit of that. So when karma runs webpack should transpile down the ES6 modules. But if it doesn't work, it doesn't work.